app/src/main/java/com/github/se/cyrcle/ui/profile/ProfileScreen.kt

```diff
@@ -41,7 +47,43 @@ fun ProfileScreen(navigationActions: NavigationActions) {
   var lastName by remember { mutableStateOf("Doe") }
   var username by remember { mutableStateOf("johndoe") }
   var profilePictureUrl by remember { mutableStateOf("") }
-  var favoriteParkings by remember { mutableStateOf(emptyList<String>()) }
+  var favoriteParkings by remember {
```

Comment:
Try to use the parkings of `TestInstances` and not harcoded values. Those test instances are more complete and it avoids hardcoding.

Reply:
I second this. It'll allow for more consistency across the board

Reply:
True, they're better then the ones I came up with. Didnt know they existed, thanks